src/test/scala/nl/knaw/dans/easy/fedoratobag/DdmSpec.scala

```diff
@@ -479,6 +479,46 @@ class DdmSpec extends TestSupportFixture with EmdSupport with AudienceSupport wi
     triedDDM.flatMap(validate) shouldBe Success(())
   }
 
+  it should "ignore empty place" in {
+    val emd = parseEmdContent(Seq(
+      emdTitle, emdCreator, emdDescription, emdDates,
+        <emd:coverage>
+          <eas:spatial>
+            <eas:place></eas:place>
```

Comment:
Please also add a test with content for place. That used to result in 

    ERROR not implemented expected either point, box or polygon [name=blablabla scheme=degrees north=90.0 east=180.0 south=-90.0 west=-180.0]

It is now silently ignored.